flask_application/predict.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+from pathlib import Path
+
+import pandas as pd
+
+path = Path(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(path)
+MODEL_PATH = os.path.join(path.parent, "datasets/modeling/")
+
+from housing_modeling.src.utils import predict  # isort:skip # noqa
+
+
+def make_prediction(dictionary={}, form=None):
```

Comment:
do not use mutable datastructures (list, dict) for default arguments. Instead you can use the following pattern
```
def make_prediction(features=None, form=None):
    features = features or {}
```

Reply:
the api is a little weird.. if you want to support one or the other, you need to check and ensure both are not passed.. instead i would just write a function to convert form data to a dictionary/dataframe and use it elsewhere.